themes/openideal_theme/openideal_theme.theme

```diff
@@ -68,16 +68,21 @@ function openideal_theme_preprocess_html(&$variables) {
  */
 function openideal_theme_preprocess_page(&$variables) {
   $route_match = \Drupal::routeMatch();
+  $route_name = $route_match->getRouteName();
   $node = $route_match->getParameter('node');
   if ($node instanceof NodeInterface) {
     $variables['node_type'] = $node->bundle();
   }
 
-  if ($route_match->getRouteName() == 'view.frontpage.front_page'
-    || $route_match->getRouteName() == 'entity.user.canonical'
-    || $route_match->getRouteName() == 'view.news.all_news_page') {
+  if ($route_name == 'view.frontpage.front_page'
```

Comment:
I would prefer to replace multiple || with the following statement:
```
$some_routes = [];
in_array($route_name, $some_routes)
```

Reply:
Done